text/0187-data-classification.md

```diff
@@ -0,0 +1,323 @@
+- Converting from non OTLP formats to OTLP would need depend on processors / converters to support this, however doing so will likely impact performance.
+- Managing classifications within the semantic convention has not be explored
+- The tolerance of what is considered a given classification could change between organisations
+
+## Prior art and alternatives
+
+There was a conversation that was related here on attributes that should be subject to data regulations Proposal: Add Sensitive Data LabelsOPEN , however, this approach would superseded what is mentioned there and be applied more broadly to the project.
+
+The alternatives that were considered are:
+
+- The Semantic Convention defines classification for known attributes
+  - this was no considered heavily since it misses USERS (developers) whom are using the SDK natively and then would need to manage their own Semantic Convention that would also need to be understood by the down stream processors and vendors
+    - Does not account for version drift with a breaking change involved
+    - Does a processor need to know all versions?
+    - Does that mean values can not be changed in the semantic convention?
+
+- Appending prefixes/suffixes to attribute keys
+  - String matching is expensive operation and would considerable slow down the current perform wins that the project strives for
+  - Can not allow for multiple definitions to be set on one attribute
+  - Has the potential to overlap with an exciting keys and require USERs to do a migration of what attributes they are already sending
+    - Unneeded work that should be avoided
+    - Limits what USERs can define their attributes keys
+  - The ownership is then moved to processors and vendors to set the definition
+    - Goes against the projects goal of being vendor neutral
+    - Each vendor processor could define different values (same as above)
+
+## Open questions
+
+Does the semantic convention define what hints are automatically set for exported attributes?
+
+Can a Service Owner change the definitions of classifications without needed to modify instrumentation code?
+
+How would this work with the semantic convention?
+
+What definitions should be used for the classifications so that the values are set subjectively?
+
+What extra work is missing to make classifications easy to manage as part of the semantic convention?
+
+## Future possibilities
+
+Some ideas of future possibilities have been eluded to throughout the proposal, the potential changes that would be of great impact are:
+
+- Vendor based classification controls
+- Vendor neutral data protection mechanisms
+- Enforcing Data Regulations on telemetry
+- Classification based routing
+  - An exporter can only with certain set of classifications
+
+Thinking more broadly, it would mean that Open Telemetry could easily extend to support analytical based telemetry on user interactions while adhering to data regulation or organisational policies.
```

Comment:
```suggestion
Thinking more broadly, it would mean that OpenTelemetry could easily extend to support analytical based telemetry on user interactions while adhering to data regulation or organisational policies.
```